fitbenchmarking/fitbenchmarking
# Fixes screen printing

#### Description of Work

Fixes #521 

Removes redundant screen printing and updating table row labels.


#### Testing Instructions

1. Check the tables do not have an extra 1 at the end of the row label.
2. Check that there are not two "Running data..." prints when running FitBenchmarking

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
